dashboard/app/views/home/_privacy.md

```diff
@@ -261,7 +261,7 @@ If any provision of this Privacy Policy or our Terms of Service is held invalid
 
 ## No Commercial Use of Student Data and How to Delete Student Data
 
-Some laws, such as California’s Student Online Personal Information Protection Act (SOPIPA), prohibit the gathering of the personal information of K-12 students. Code.org abides by such laws and shall not use, disclose, or compile any student personal information for the purpose of marketing or advertising commercial products or services. We do not disclose any Personal Information to third parties for marketing purposes. You may request to have your Personal Information deleted from the Code.org database at any time.
+Some laws, such as California’s Student Online Personal Information Protection Act (SOPIPA), prohibit the gathering of the Personal Information of K-12 students. Code.org abides by such laws and shall not use, disclose, or compile any student Personal Information for the purpose of marketing or advertising commercial products or services. We do not disclose any student Personal Information to third parties for marketing purposes. A student may request to have their Personal Information deleted from the Code.org database at any time.
```

Comment:
Should these three `Student` be capitalized?

Reply:
No, we want it to be broader than students who have accounts.